lib/datura/options.rb

```diff
@@ -2,20 +2,26 @@
 
 require_relative './helpers.rb'
 
-class Options
+class Datura::Options
   attr_reader :all
 
-  def initialize(params, general_config_path, collection_config_path)
-    @directory = File.dirname(__FILE__)
-    @collection_dir = params["collection_dir"]
+  def initialize(params)
     @environment = params["environment"]
+    # paths to the collection and gem directories
+    collection_dir = Dir.getwd
```

Comment:
That's true, you would have to run it at the base of the repo.  I kind of imagine that a lot of times these won't be stored under the path `.../collections/` because ostensibly you could put these repositories anywhere and in any sort of setup, like when you're working on them locally.  And I've uncoupled the `collection` parameter with the collection directory's name because of problems in the past with things like `cather-complete-letters` and `complete-letters` needing to post to the "cather" index...hmm.

I could instead pull the current file's directory and navigate backwards to get to the base of the repository.  That's how the gem used to work in the past, I was just ever so pleased about having these little autonomous directories that I wanted to just use that instead!  :)

What do you think?

Reply:
I think it's probably better to only be able to call these scripts from root, but I could be convinced otherwise. If anything, this should be an issue to discuss , I don't think it needs to hold up this PR.

Reply:
Jess, I hadn't thought about it being checked out to arbitrary places where there won't be a parent `collections/` directory, but your idea of navigating backwards would get to the same intended result. Yeah, this can be an issue for later.

Reply:
https://github.com/CDRH/data/issues/131